Packages/com.unity.inputsystem/InputSystem/Plugins/Android/AndroidSensors.cs

```diff
@@ -115,7 +116,7 @@ public FourCC GetFormat()
         }
     }
 
-    public class AndroidSensorProcessor : IInputControlProcessor<Vector3>
+    public class AndroidCompensateDirectionProcessor : CompensateDirectionProcessor
```

Comment:
Ideally, should indicate functional difference to base processor by its name (or at least comment).